discussion/app/AppLoader.scala

```diff
@@ -1,13 +1,13 @@
 import dev.DevAssetsController
 import http.CorsHttpErrorHandler
-import app.{FrontendComponents, FrontendApplicationLoader}
+import app.{FrontendApplicationLoader, FrontendComponents}
 import com.softwaremill.macwire._
 import common._
 import common.Logback.LogstashLifecycle
 import conf.switches.SwitchboardLifecycle
-import conf.{CommonFilters, CachedHealthCheckLifeCycle}
+import conf.{CachedHealthCheckLifeCycle, CommonFilters}
 import controllers.{DiscussionControllers, HealthCheck}
-import discussion.DiscussionApi
+import discussion.api.DiscussionApi
```

Comment:
Not related but moving the `DiscussionApi` class in `discussion.api` package, where it belongs imho